dcmio/dcmconverter/dicom_utils.py

```diff
@@ -99,10 +100,22 @@ def add_meta_to_nii(nii_files, dicom_dir, prefix, dcm_tags, output_directory,
                 header.set_slice_duration(slice_duration)
 
             # > add free dicom fields
-            content = dict(
-                (str(name), str(dcmimage[tag].value))
-                for name, tag in dcm_tags)
-
+            # extract value from the dicom file
+            content = {}
+            for name, tag in dcm_tags:
+
+                try:
+                    # enhances storage, the value is burried under one or
+                    # several layer(s) of sequence
+                    dcmimage_temp = dcmimage
+                    if len(tag) > 1:
+                        for sub_tag in tag[:-1]:
+                            seq_field = dcmimage_temp[sub_tag]
+                            dcmimage_temp = seq_field.value[0]
```

Comment:
data_element = dcmimage_temp[inner_tag]
Need to test VR?
if data_element.VR == "SQ":
    Is it sufficient to keep the first inner dataset?
    current_dataset = data_element.value[0]
else:
    raise something